crates/sui-tool/src/bin/cli-tool.rs

```diff
@@ -6,8 +6,7 @@ use clap::*;
 use colored::Colorize;
 use sui_types::exit_main;
 
-mod commands;
-use commands::ToolCommand;
+use sui_tool::commands::ToolCommand;
```

Comment:
Can we keep the output binary name `sui-tool`? It is clear and not generic. There are many scripts and workflows depending on the name.

Also people may copy the binary to their PATH. It will be inconvenient to ask people to rename after building the binary.